### PR TITLE
Proofread

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -998,16 +1182,26 @@ RFC: [Remove deprecated functionality in PHP 7](https://wiki.php.net/rfc/remove_
...
 RFC: [Reclassify E_STRICT notices](https://wiki.php.net/rfc/reclassify_e_strict)
 
 ### Deprecation of Salt Option for `password_hash()`
 
-With the introduction of the new password hashing API in PHP 5.5, many began implementing it and generating their own salts. Unfortunately, many of these salts were generated from cryptographically insecure functions like mt_rand(), making the salt far weaker than what would have been generated by default. (Yes, a salt is always when hashing passwords with this new API!) The option to generate salts have therefore been deprecated to prevent developers from creating insecure salts.
+With the introduction of the new password hashing API in PHP 5.5, many began
+implementing it and generating their own salts. Unfortunately, many of these
+salts were generated from cryptographically insecure functions like mt_rand(),
+making the salt far weaker than what would have been generated by default.
+(Yes, a salt is always when hashing passwords with this new API!) The option to
```
Comment (samvaughton) [inline README.md]: Missing a word, it should be "...a salt is always **used** when hashing passwords..."

Comment (greg0ire) [inline README.md]: You're right. Can you make a P.R. on my branch ?

Comment (samvaughton) [inline README.md]: Done

**Comment by tpunt** [comment]
Thanks for the changes.

**Pull request merged**